.github/workflows/status-check-override.yaml

```diff
@@ -0,0 +1,54 @@
+# For pull requests that only modify documentation, individual CI events may forego running.
+# (This is done by each action/pipeline/event's config.) The itwinjs-core repo requires some
+# of those events to have finished successfully or neutrally to allow a pull request to be
+# merged.
+#
+# This config will fake successful status checks for such events when it detects
+# *only* documentation changes in a pull request.
```

Comment:
Maybe we should make a note here that it is not necessary to fake the ADOps jobs that are skipped due to path filtering, and this is only needed for the checks ran in GH actions?

Reply:
Good suggestion. How does the comment I just added sound?